source/documentation/runbooks/004-analytical-platform-compute-maintenance.html.md.erb

```diff
@@ -0,0 +1,117 @@
+---
+owner_slack: "#analytical-platform-notifications"
+title: Analytical Platform Compute Maintenance - Draft
+last_reviewed_on: 2024-10-28
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+
+On the first day of the month the workflow [schedule-issue-compute-infrastructure.yml] (https://github.com/ministryofjustice/analytical-platform/blob/bba07f4908b443a2c15bf94738c8b84c5211e9f9/.github/workflows/schedule-issue-compute-infrastructure.yml) will automatically raise a ticket for [Maintenance - Analytical Platform Compute ] (https://github.com/ministryofjustice/analytical-platform/issues/5661 ).
+
+This maintenance ticket includes EKS Cluster Upgrade if applicable and/or patching to ensure all components are up to date.
+
+**Note:** Currently these tickets need manually pulling into the sprint.
+
+##  Check for new release
+
+Check if a new release of Amazon EKS Kubernetes has been made available [here](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar
+).
+
+If there is a new EKS release available carry out EKS upgrade and patch all other components.
+
+Or
+
+If there is not a new EKS release available patch all other components including addons.
+
+## The Approach
+
+1. Create a new branch / PR.
```

Comment:
I personally adopt a long form approach in formal written scenarios like this, though "PR" is acceptable I would use "Pull Request" preferably. In line 32, this is the case. First and foremost if an abbreviation is adopted it should be used consistently from that point forward. 

TLDR I'd alter this section to use "Pull Request" throughout.

Reply:
amended to Pull Request